src/tools/info/ucx_info.c

```diff
@@ -52,6 +55,7 @@ static void usage() {
     printf("                    'shm'  : shared memory devices only\n");
     printf("                    'net'  : network devices only\n");
     printf("                    'self' : self transport only\n");
+    printf("  -A <ip_address> IP address of the server to connect to\n");
```

Comment:
`-A <ipv4>   Local network device address to use for creating endpoint in client/server mode`